config/kernel-monitor-filelog.json

```diff
@@ -9,6 +9,7 @@
 	"lookback": "5m",
 	"bufferSize": 10,
 	"source": "kernel-monitor",
+	"skipList": [ " audit:", " audit[" ],
```

Comment:
Can you remove from this file and add a new json file with this?

Reply:
done